content/android/get-started.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Get Started"
+date: 2017-07-03T22:20:00-04:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "android"
+projectname: "WorldWind Android"
+listdescription: ""
+layout: "project"
+---
+
+## Getting Started
+
+WorldWind Android is an [Android Library](https://developer.android.com/studio/projects/android-library.html) which compiles into an Android Archive (AAR) file. The AAR can be linked and used to display the 3D virtual globe within your application. The WorldWind Android AAR is compatible to API level 19 (Android 4.4 KitKat).
+
+Skip to the [**Download**](#download) section for directions on how to add WorldWind Android to your application. If you need some guidance with setting up the Android Studio IDE, the [**Setup**](#setup) section includes in-depth directions and walks through the creation of a sample project utilizing the WorldWind library.
+
+---
+### <a name="download"></a>Download
+
+For [Gradle](https://gradle.org) based projects, versioned WorldWind Android releases are hosted on [JCenter](https://bintray.com/bintray/jcenter). Grab the latest version by adding the following to the `dependencies` closure in your `build.gradle` file:
+```groovy
+compile 'gov.nasa.worldwind.android:worldwind:{{% latestBintrayVersion url="https://api.bintray.com/packages/nasaworldwind/maven/WorldWindAndroid/versions/_latest" %}}'
+```
+
+To manually download the (AAR) and other artifacts (source and javadocs) visit: https://bintray.com/nasaworldwind/maven/WorldWindAndroid
+
+`SNAPSHOT` builds are available through [OJO](https://oss.jfrog.org/). Add the following repository to your Gradle project:
+```groovy
+repositories {
+    maven {
+        url 'https://oss.jfrog.org/artifactory/oss-snapshot-local'
+    }
+}
+```
+Add the latest `SNAPSHOT` WorldWind Android dependency:
+```groovy
+compile 'gov.nasa.worldwind.android:worldwind:{{% latestOjoVersion url="https://oss.jfrog.org/artifactory/api/search/versions?g=gov.nasa.worldwind.android&a=worldwind&repos=oss-snapshot-local" %}}'
+```
+---
+### <a name="setup"></a>Setup
+#### Install Android Studio
+[Android Studio](https://developer.android.com/studio/index.html) is the IDE used by the WorldWind development team. It includes all the tools you need to build apps for Android. If you haven't done so already, download and install Android Studio.
+
+{{< button url="https://developer.android.com/studio/index.html" buttonText="Get Android Studio" >}}
```

Comment:
I think we ought to reserve this button image for our own links. It does look good but my first impression was that we (NASAWorldWind) had something to do with installing Android Studio.